.changeset/short-timers-return.md

```diff
@@ -0,0 +1,9 @@
+---
+"@comet/cms-site": minor
+"@comet/cms-api": minor
+---
+
+Use more generic resolver to support prelogin
+
+-   `@comet/cms-api`: Provide a `currentUser` field resolver `permissionsForScope` instead the `isAllowedSitePreview`-query.
```

Comment:
The `isAllowedSitePreview` query would be added in v7, do we even need this changeset?